test/testGitUtils.js

```diff
@@ -82,7 +82,12 @@ describe('Testing GitUtils', () => {
     assert.equal(await GitUtils.isDirty(testRoot, GIT_USER_HOME), true);
   });
 
-  it('isDirty #unit with submodules', async () => {
+  it('isDirty #unit with submodules', async function test() {
```

Comment:
https://github.com/adobe/helix-testutils can help with that.

Reply:
my IDE doesn't llike the `condit`... ;-)

Reply:
but the problem is, when the test is skipping itself, the `afterEach` is not executed anymore; which can lead to side effects...

Reply:
see https://github.com/mochajs/mocha/issues/3740